server/document.js

```diff
@@ -97,11 +97,14 @@ export class NextScript extends Component {
 
   render () {
```

Comment:
@bringking I tested with very low network connections and script async do fine.
Since link preload only chrome now and script async is doing great for us, I'm going to defer this to later.